src/audio-player-composable/components/forward-button/forward-button.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import {IconButton} from '../../../icon-button';
+import {ButtonSize} from '../../../button/types';
+import {useAudioPlayerContext} from '../../context';
+import {IconFilledForward10} from '../../../icons';
+import {SkipButtonProps} from '../replay-button/types';
+
+export const AudioPlayerForwardButton: React.FC<SkipButtonProps> = React.memo(
+  ({onClick: consumerOnClick, ...props}) => {
+    const {getForwardButtonProps} = useAudioPlayerContext();
+    const {ariaLabel, onClick} =
+      getForwardButtonProps! &&
+      getForwardButtonProps({onClick: consumerOnClick});
+
+    return (
+      <IconButton
+        data-testid="audio-player-forward-button"
+        aria-label={ariaLabel}
+        onClick={onClick}
+        size={props.size || ButtonSize.Medium}
+        overrides={props.overrides}
+        {...props}
+      >
+        <IconFilledForward10 />
+      </IconButton>
+    );
```

Comment:
idealy you should be able to do something like this
```suggestion
    const propsFromContex =
      getForwardButtonProps! &&
      getForwardButtonProps({onClick: consumerOnClick});

    return (
      <IconButton
        data-testid="audio-player-forward-button"
        size={props.size || ButtonSize.Medium}
        {...propsFromContex}
        {...props}
      >
        <IconFilledForward10 />
      </IconButton>
    );
```